apps/web/src/pages/templates/components/email-editor/EmailContentCard.tsx

```diff
@@ -63,16 +67,13 @@ export function EmailContentCard({
 
   return (
     <>
-      {!isIntegrationActive ? (
+      {!isIntegrationActive && isLimitReached && (
         <LackIntegrationError
-          channelType="E-Mail"
-          text={
-            enabled
-              ? 'Looks like you haven’t configured your E-Mail provider yet, visit the integrations page to configure.'
-              : undefined
-          }
+          channel="E-Mail"
+          channelType={ChannelTypeEnum.EMAIL}
+          text="Looks like you haven’t configured your E-Mail provider yet, visit the integrations page to configure."
```

Comment:
Not sure if this error message aligns with the attribute `isLimitReached`. Also will it show for non active integrations without any limit?
Overall I am unable to understand properly the functionalities here (already from the previous version). 😕

Reply:
This banner will be only shown if there is no active email integration and the user has reached the Novu provider credits. In other words, we won't show it when: the user has Novu credits or has active email integration. Because it means that he has an email provider.